data/_models/shared/Collection events.csv

```diff
@@ -1,9 +1,11 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,visible
-Collection events,,,,,,,,,,,dcterms:PeriodOfTime,Definition of a data collection event for a resource,DataCatalogueFlat,
-Collection events,,collection,ref,1,true,,Collections,,,,dcterms:isPartOf,Resource this collection event is part of,DataCatalogueFlat,
-Collection events,,name,,1,true,,,,,,dcterms:title,Name of the collection event,DataCatalogueFlat,
-Collection events,,description,text,,,,,,,,dcterms:description,Description of the collection event,DataCatalogueFlat,
-Collection events,,populations,ref_array,,,,Collection populations,collection,,,dcterms:isReferencedBy,Subcohorts that are targetted by this collection event,DataCatalogueFlat,
+Collection events,,,,,,,,,,,dct:PeriodOfTime,Definition of a data collection event for a resource,DataCatalogueFlat,
```

Comment:
I think these should be the dcat:DatasetSeries. See the Health-RI wiki "With Dataset Series we refer to data, somehow interrelated, that are published separately. An example is budget data split by year and/or country, instead of being made available in a single dataset." https://health-ri.atlassian.net/wiki/spaces/FSD/pages/121110529/Core+Metadata+Schema+Specification#Recommended-Classes